### PR TITLE
[SELC-6089] Feat: Integrate mobile phone in edit user info

### PR DESCRIPTION
**Pull request by OraldoDoci**
#### List of Changes
Add input field for mobile phone
update swagger
on redirect make phone number in acitve field

#### Motivation and Context
Account needs to contact the administrators of the instituions

#### How Has This Been Tested?
Tested locally and by mocking dev enviroment

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: openApi/dashboard-api-docs.json [hunk 1]
```diff
@@ -3938,7 +3942,7 @@
           },
           "selcRole" : {
             "type" : "string",
-            "description" : "User's role, available value: [MANAGER, DELEGATE, SUBDELEGATE, OPERATOR, ADMIN_EA]",
+            "description" : "User's Selfcare role, available value: [ADMIN, ADMIN_EA, LIMITED]",
```
Comment (AleDore) [inline openApi/dashboard-api-docs.json:3945]:
```suggestion
            "description" : "User's Selfcare role, available values: [ADMIN, ADMIN_EA, LIMITED]",
```

File: openApi/dashboard-api-docs.json [hunk 2]
```diff
@@ -4031,7 +4035,7 @@
           },
           "role" : {
             "type" : "string",
-            "description" : "User's role, available value: [MANAGER, DELEGATE, SUBDELEGATE, OPERATOR, ADMIN_EA]",
+            "description" : "User's Selfcare role, available value: [ADMIN, ADMIN_EA, LIMITED]",
```
Comment (AleDore) [inline openApi/dashboard-api-docs.json:4038]: ditto

File: openApi/generated-dashboard/dashboard-swagger20.json [hunk 3]
```diff
@@ -3550,7 +3554,7 @@
           "type": "string"
         },
         "selcRole": {
-          "description": "User's role, available value: [MANAGER, DELEGATE, SUBDELEGATE, OPERATOR, ADMIN_EA]",
+          "description": "User's Selfcare role, available value: [ADMIN, ADMIN_EA, LIMITED]",
```
Comment (AleDore) [inline openApi/generated-dashboard/dashboard-swagger20.json:3557]:
```suggestion
          "description": "User's Selfcare role, available values: [ADMIN, ADMIN_EA, LIMITED]",
```

File: openApi/generated-dashboard/dashboard-swagger20.json [hunk 4]
```diff
@@ -3651,7 +3655,7 @@
           "description": "Authorized user product"
         },
         "role": {
-          "description": "User's role, available value: [MANAGER, DELEGATE, SUBDELEGATE, OPERATOR, ADMIN_EA]",
+          "description": "User's Selfcare role, available value: [ADMIN, ADMIN_EA, LIMITED]",
```
Comment (AleDore) [inline openApi/generated-dashboard/dashboard-swagger20.json:3658]:
```suggestion
          "description": "User's Selfcare role, available values: [ADMIN, ADMIN_EA, LIMITED]",
```

File: src/api/DashboardApiClient.ts [hunk 5]
```diff
@@ -132,9 +132,10 @@ export const DashboardApi = {
       institutionId,
       id: user.id,
       body: {
-        email: user.certifiedMail ? undefined : user.email,
```
Comment (AleDore) [inline src/api/DashboardApiClient.ts:135]: why not consider certified email?

Comment (OraldoDoci) [reply src/api/DashboardApiClient.ts:135]: Request by BE to always send the mail.

**Review by AleDore — APPROVED** [review]

**Pull request merged**